integration/app/test-demo/README.md

```diff
@@ -0,0 +1,76 @@
+# README - React Testing Demo
+
+The React Testing Demo is used to test the Amazon Chime SDK React Components Library.
```

Comment:
Are we calling it React testing Demo or Mini test Apps?

Reply:
The demo itself is called React Testing Demo.
Those mini test Apps are called "Mini Test App", e.g., Roster Test

Reply:
Oh then can you make that clear here - "The React Testing Demo is comprised of multiple Mini Test Apps"

Reply:
Yeah I mentioned it here:
>Following the example of [WebRTC samples](https://webrtc.github.io/samples/), we come up with the concept of a Mini Test App - a minimalist, single-page application containing only the necessary components for a single test. **All Mini Test Apps are sub-pages of the React Testing Demo.**